webpack/assets/javascripts/react_app/components/common/AutoComplete/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Downshift from 'downshift';
+import classnames from 'classnames';
+import { InputGroup } from 'react-bootstrap';
+import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import './AutoComplete.scss';
+
+const AutoComplete = ({
+  onSelect, labelText, onInputUpdate, items,
+}) => (
+  <Downshift
+    onChange={onSelect}
+    itemToString={i => (i == null ? '' : String(i))}
+    render={({
+      getInputProps,
+      getItemProps,
+      getLabelProps,
+      isOpen,
+      inputValue,
+      highlightedIndex,
+      getRootProps,
+      selectedItem,
+    }) => (
+      <div>
+        {labelText ? <label {...getLabelProps()}>{labelText}</label> : null}
+        <InputGroup>
+          <input
+            type="text"
+            className="form-control"
+            {...getInputProps({
+              onChange: ({ target: { value } }) => {
+                onInputUpdate(value);
+              },
+            })}
+          />
+          <InputGroup.Button>
+            <Button onClick={() => onSelect(inputValue)}>Search</Button>
+          </InputGroup.Button>
+        </InputGroup>
+
+        {isOpen && items.length ? (
+          <div className="autocomplete-dropdown-container">
```

Comment:
I think you can use `DropdownMenu` and `MenuItem` from `patternfly-react`, then you should be able to remove your `scss` file
https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?knob-Disabled=false&knob-Style=default&selectedKind=MenuItem&selectedStory=MenuItem&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs